cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java

```diff
@@ -57,8 +83,13 @@
     public static final String GITIGNORE = "gitignore";
     public static final String NEW_CMD_DEFAULTS = "new_cmd_defaults";
     public static final String CREATE_CMD_TEMPLATES = "create_cmd_templates";
+    protected static PrintStream printStream;
```

Comment:
Does this particular printStream have any use?

Reply:
Removed